### ISSUE TITLE
Suggestion: a hotkey to exit the program completely

### ISSUE THREAD

**Issue by mikkovedru** [KryDos/todoist-linux#26]
There might be situations when I would like to close the program (like me playing a resource-heavy game, installing a new version of the todoist-linux, having too little memory and needing just the one that Todoist-linux's Electron utilizes).

Would be nice to have an option to exit the program completely using the internal hotkey/button. Currently the only way is to kill it via System Monitor. There should be a cleaner way, imho.

**Comment by mikkovedru** [comment]
> I'll think about it and probably remove the "work in background" mode completely so when you close the app the process will be killed.
Please don't do it. This is the only reason I use this app.

When it comes to tray functionality, I remembered it only after you wrote about it. I use collapsible tray and completely forgot that. But now I can confirm that exiting through the tray icon works fine - it closes the whole program as it should.

So that closes the issue of having a clean way to close the program.

But that still leaves the issue of a specific hotkey combination to close the program. Currently Alt-F4 closes it to tray. Clicking the X in the corner of the app also closes it to tray (this functionality should not be changed!).

If it's not too difficult, maybe you could establish another hotkey for closing the app completely (preferably with confirmation dialog; if possible, add this dialog also to the tray->exit action)?

This is not a must-have feature for me. Tray functionality is enough for me. This is just an idea for you to consider and implement if you have time, energy and will.

**Comment by KryDos** [comment]
hi @mikkovedru,
there is new shortcut added to quit the app completely without hiding it into Tray. 
The shortcut is `Alt-F4`. 
I wanted to use `Ctrl+Alt+q` but it's already bind.

You can find that change in [the latest build](https://github.com/KryDos/todoist-linux/releases/tag/1.14)
